<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>英雄联盟之英雄类型表-专业软件定制开发 https://xinyiya.taobao.com/</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>选择英雄</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="txtHeroSearch"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnCancelAllSelect">
            <property name="text">
             <string>取消全选</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="lstHeros"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>英雄类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tblHeros"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="text">
       <string>鑫意雅淘宝店: https://xinyiya.taobao.com/</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>commandLinkButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>linkMyShop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelAllSelect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cancelAllSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtHeroSearch</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>heroSearchChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>linkMyShop()</slot>
  <slot>cancelAllSelected()</slot>
  <slot>heroSearchChanged()</slot>
 </slots>
</ui>
